docker-compose.yml

```diff
@@ -91,7 +91,8 @@ services:
         condition: service_healthy
     platform: linux/amd64
     volumes:
-      - .:/usr/src/app
+      - ./src:/usr/src/app/src
+      - ./style:/usr/src/app/style
```

Comment:
These are really the only folders that can be watched for hot-reloading. Mounting everything was causing the .env file (which gets copied in the build step, see the Dockerfile) to go missing.